pkgs/applications/misc/furtherance/default.nix

```diff
@@ -30,13 +31,7 @@ stdenv.mkDerivation rec {
     wrapGAppsHook4
   ];
 
-  buildInputs = [
-    dbus
-    glib
-    gtk4
-    libadwaita
-    sqlite
-  ];
+  buildInputs = [ dbus glib gtk4 libadwaita sqlite ];
```

Comment:
Why this format change?

Reply:
Sorry about that, forgot to untoggle the autoformatter.